### PR TITLE
Disable iOS OTA Updates and Hide OTA Badge

### PR DESCRIPTION
**Pull request by ijapija00**

### CODE REVIEW

**Comment by tangalbert919** [comment]
@ijapija00 is here! :smile:

**Comment by coolstar** [comment]
This commit needs serious cleanup before it can be considered for merge. There's a ton of changes that were marked by git that are irrelevant to the commit. Please keep the diff short and only keep the lines that are relevant.

File: yalu102/jailbreak.m [hunk 1]
```diff
@@ -474,30 +474,30 @@ void exploit(void* btn, mach_port_t pt, uint64_t kernbase, uint64_t allprocs)
...
-        copyin(bbuf, TTE_GET(tte, TTE_PHYS_VALUE_MASK) - gPhysBase + gVirtBase, PSZ);\
-        copyout(newt, bbuf, PSZ);\
-        TTE_SET(tte, TTE_PHYS_VALUE_MASK, findphys_real(newt));\
-        TTE_SET(tte, TTE_BLOCK_ATTR_UXN_MASK, 0);\
-        TTE_SET(tte, TTE_BLOCK_ATTR_PXN_MASK, 0);\
-        WriteAnywhere64(tte_addr, tte);\
-        NSLog(@"level %llx - %llx", tte_addr,              TTE_GET(tte, TTE_PHYS_VALUE_MASK));\
-    }, level1_table, 2);
-
+pagestuff_64((address) & (~PMK), ^(vm_address_t tte_addr, int addr) {\
```
Comment (coolstar) [inline yalu102/jailbreak.m]: Don't mess with the indentation in the code, as it's irrelevant to the change you are describing.

File: yalu102/jailbreak.m [hunk 2]
```diff
@@ -710,7 +710,22 @@ void exploit(void* btn, mach_port_t pt, uint64_t kernbase, uint64_t allprocs)
...
-
+    
+    { //Prevents OTA Daemons from launching && Removes OTA Badges
+        
+        int f1 = open("/System/Library/LaunchDaemons/com.apple.mobile.softwareupdated.bak", O_RDONLY);
+        int f2 = open("/System/Library/LaunchDaemons/com.apple.softwareupdateservicesd.bak", O_RDONLY);
+        
+        if ( (f1 == -1) || (f2 == -1) )  {
+            NSLog(@"Killing OTA Updates...");
+            system("mv -f /System/Library/LaunchDaemons/com.apple.mobile.softwareupdated.plist /System/Library/LaunchDaemons/com.apple.mobile.softwareupdated.bak");
```
Comment (coolstar) [inline yalu102/jailbreak.m]: Don't shell out to mv

**Comment by KILLCAMPER** [comment]
That's hope he can help with sum 4K device compatible issues

**Comment by tangalbert919** [comment]
@coolstar is also here?
Are you two contributing to this jailbreak? I'd ask if YUCCA Utilities will work with this, but ti's better if I don't.

**Comment by coolstar** [comment]
White space should not be unnecessarily truncated in the same commit as other changes. Keep the commit clean. If you want to do formatting changes, do it in a separate commit and pull request.

**Pull request closed without merging**